be/src/Project/Services/Inventory/Inventory.Api/Infrastructure/Entities/Outlet.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Foundation.GenericRepository.Entities;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Inventory.Api.Infrastructure.Entities
+{
+    public class Outlet : Entity
+    {
+        [Required]
+        public Guid? ProductId { get; set; }
```

Comment:
**suggestion:** nullable Guid makes no sense with required attribute. It should be:
[Required] 
public Guid ProductId { get; set; }